packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts

```diff
@@ -94,12 +94,18 @@ export const peoplePaneContainerTokens: IStackTokens = {
 /**
  * @private
  */
-export const participantListWrapper = mergeStyles({
-  width: '20rem',
-  // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
-  maxWidth: '50vw',
-  height: '100%'
-});
+export const participantListWrapper = (theme: Theme): string =>
+  mergeStyles({
+    width: '20rem',
+    // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
+    maxWidth: '50vw',
```

Comment:
we can't use `vw` when styling as this is 50% of the browser width, and not 50% of the chatcomposite's parent element (the chat composite won't always be fully spanning the screen, and when it isn't this css won't function as expected)

Reply:
Going by the figma this looks to be taking up a lot more than 50%:
![image](https://user-images.githubusercontent.com/2684369/171278321-dac3c33e-698b-402e-aad0-b8c9133986e4.png)

Reply:
I have changed it from 50vw to 50%, behaviour is same. Let me know if that works or should we use fixed width?